### DISCUSSION TITLE
docker-compose 安装 -【数据初始化】遇错

### DISCUSSION THREAD

**Discussion by zangjietutu** [hhyo/Archery#2190] [Q&A] [answered]
![image](https://github.com/hhyo/Archery/assets/53146865/d4baa210-cb4b-41a7-b82d-564b0b56689d)

执行以下命令报错
python3 manage.py dbshell<sql/fixtures/auth_group.sql

**Comment by nick2wang** [comment, marked as answer]
切换到虚拟环境了么？
```
source /opt/venv4archery/bin/activate
```

**Reply by zangjietutu** [reply]
进入 容器后  执行 source /opt/venv4archery/bin/activate 进入虚拟环境？
